packages/babel-runtime-corejs2/package.json

```diff
@@ -18,5 +18,265 @@
+    "./helpers/esm/toArray": "./helpers/esm/toArray.js",
+    "./helpers/esm/toConsumableArray": "./helpers/esm/toConsumableArray.js",
+    "./helpers/esm/arrayWithoutHoles": "./helpers/esm/arrayWithoutHoles.js",
+    "./helpers/esm/arrayWithHoles": "./helpers/esm/arrayWithHoles.js",
+    "./helpers/esm/maybeArrayLike": "./helpers/esm/maybeArrayLike.js",
+    "./helpers/esm/iterableToArray": "./helpers/esm/iterableToArray.js",
+    "./helpers/esm/iterableToArrayLimit": "./helpers/esm/iterableToArrayLimit.js",
+    "./helpers/esm/iterableToArrayLimitLoose": "./helpers/esm/iterableToArrayLimitLoose.js",
+    "./helpers/esm/unsupportedIterableToArray": "./helpers/esm/unsupportedIterableToArray.js",
+    "./helpers/esm/arrayLikeToArray": "./helpers/esm/arrayLikeToArray.js",
+    "./helpers/esm/nonIterableSpread": "./helpers/esm/nonIterableSpread.js",
+    "./helpers/esm/nonIterableRest": "./helpers/esm/nonIterableRest.js",
+    "./helpers/esm/createForOfIteratorHelper": "./helpers/esm/createForOfIteratorHelper.js",
+    "./helpers/esm/createForOfIteratorHelperLoose": "./helpers/esm/createForOfIteratorHelperLoose.js",
+    "./helpers/esm/skipFirstGeneratorNext": "./helpers/esm/skipFirstGeneratorNext.js",
+    "./helpers/esm/toPrimitive": "./helpers/esm/toPrimitive.js",
+    "./helpers/esm/toPropertyKey": "./helpers/esm/toPropertyKey.js",
+    "./helpers/esm/initializerWarningHelper": "./helpers/esm/initializerWarningHelper.js",
+    "./helpers/esm/initializerDefineProperty": "./helpers/esm/initializerDefineProperty.js",
+    "./helpers/esm/applyDecoratedDescriptor": "./helpers/esm/applyDecoratedDescriptor.js",
+    "./helpers/esm/classPrivateFieldLooseKey": "./helpers/esm/classPrivateFieldLooseKey.js",
+    "./helpers/esm/classPrivateFieldLooseBase": "./helpers/esm/classPrivateFieldLooseBase.js",
+    "./helpers/esm/classPrivateFieldGet": "./helpers/esm/classPrivateFieldGet.js",
+    "./helpers/esm/classPrivateFieldSet": "./helpers/esm/classPrivateFieldSet.js",
+    "./helpers/esm/classPrivateFieldDestructureSet": "./helpers/esm/classPrivateFieldDestructureSet.js",
+    "./helpers/esm/classStaticPrivateFieldSpecGet": "./helpers/esm/classStaticPrivateFieldSpecGet.js",
+    "./helpers/esm/classStaticPrivateFieldSpecSet": "./helpers/esm/classStaticPrivateFieldSpecSet.js",
+    "./helpers/esm/classStaticPrivateMethodGet": "./helpers/esm/classStaticPrivateMethodGet.js",
+    "./helpers/esm/classStaticPrivateMethodSet": "./helpers/esm/classStaticPrivateMethodSet.js",
+    "./helpers/esm/decorate": "./helpers/esm/decorate.js",
+    "./helpers/esm/classPrivateMethodGet": "./helpers/esm/classPrivateMethodGet.js",
+    "./helpers/esm/classPrivateMethodSet": "./helpers/esm/classPrivateMethodSet.js",
+    "./helpers/esm/wrapRegExp": "./helpers/esm/wrapRegExp.js",
+    "./package.json": "./package.json",
+    "./regenerator": "./regenerator/index.js",
+    "./regenerator/": "./regenerator/",
+    "./core-js/": "./core-js/",
```

Comment:
Note that the directory export map is _deep_: All files under `core-js` can be imported via `path/filename.ext`:
```js
import "@babel/runtime-corejs2/core-js/object/assign.js"
```